index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+</head>
+
+<body>
+    <!-- HEADER -->
+    <div class="bodydiv">
+        <div class="header">
+            <!-- LOGS -->
+            <div class="sigub-box">
+                <div class="forget-password">
+                    <h5 class="htag">Already have an Intuit account? <br>
+                        <a class="ahref" href=""> I forgot my User ID or Password </a> </h5>
+
+                </div>
+                <div>
+                    <button class="button">
+                        <i class="fa fa-lock" aria-hidden="true"></i> Sign Up
+                    </button>
+                </div>
+            </div>
+        </div>
+        <!-- THE FORM DIV STARTS HERE -->
+        <div class="contain-div">
+            <div class="main-container">
+                <div class="header2">
+                    <div class="first-log">
+                        <div class="logo">
+                            <a href=""> <img src="images/1.png" alt=""> </a>
+                        </div>
+                    </div>
+                    <div class="second-logs">
+                        <div class="logo2">
+                            <a href=""> <img src="images/2.png" alt=""> </a>
+                            <a href=""> <img src="images/3.png" alt=""> </a>
+                            <a href=""> <img src="images/4.png" alt=""> </a>
+                        </div>
+                    </div>
+
+                </div>
+
+                <!-- FORM START HERE -->
+                <div class="form-container">
+                    <div class="heading-form">
+                        <h2>Create an Intuit account </h2>
+                        <p>One account for everything Intuit, including Mint. <br> <a href=""> Learn more</a> </p>
+                    </div>
+                    <div class="main-form">
+                        <form>
+                            Email address:<br>
+                            <input type="email" name="username"><br>
+                            Phone (recommended):<br>
+                            <input type="number" name="no"> <br>
+                            Standard call, messaging or data rates may apply. <br> <br>
+                            Password:<br>
```

Comment:
instead of the lines you have add like `Email address: <br>` , you should use label for that.